packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -333,26 +333,44 @@ function attachSinkId(element, sinkId) {
   }
 }
 
+function playVideoIfPaused(videoElem) {
+  if (!videoElem || !(videoElem instanceof HTMLVideoElement)) {
+    return;
+  }
+  if (videoElem.paused || videoElem.ended) {
+    videoElem.play();
+  }
+}
+
```

Comment:
Clean implementation of this logic. Could you add some comments around these if statements providing why this is necessary within the sample?

```js
// Separate logic for Safari prevents X scenario.
```

Reply:
Sure np, I'll update.